bigchaindb/models.py

```diff
@@ -71,14 +41,16 @@ def validate(self, bigchain):
             if inputs_defined:
                 raise ValueError('A CREATE operation has no inputs')
             # validate asset
-            self.asset._validate_asset()
+            amount = sum([condition.amount for condition in self.conditions])
```

Comment:
repeating earlier https://github.com/bigchaindb/bigchaindb/pull/794#discussion_r87822113:

@r-marques pointed out that although it may be more memory efficient, it would be slower. Small tests show that to be true:

```python
>>> timeit.timeit('sum(n for n in range(10000))', number=10000)
6.362047109985724

>>> timeit.timeit('sum([n for n in range(10000)])', number=10000)
4.701008878997527

>>> timeit.timeit('sum((n for n in range(10000)))', number=10000)
6.287681774992961
```